docs/docsite/rst/user_guide/playbooks_tests.rst

```diff
@@ -109,6 +109,41 @@ To see if a list includes or is included by another list, you can use 'subset' a
             msg: "B is included in A"
           when: b is subset(a)
 
+.. _contains_test:
+
+Test if a list contains a value
+```````````````````````````````
+
+.. versionadded:: 2.8
+
+Ansible includes a ``contains`` test which operates similarly, but in reverse of the Jinja2 provided ``in`` test.
```

Comment:
This entry assumes the reader knows how the `in` test works. Do we document that anywhere? If so, provide a link. If not, a short compare/contrast would be helpful here.

Reply:
its documented on the Jinja2 site

Reply:
ah, yeah, I see it at http://jinja.pocoo.org/docs/2.10/templates/#in - not much detail there - we can leave out the link for now and I'll think about ways to improve the tests documentation more generally